admin.php

```diff
@@ -25,10 +25,11 @@
 }
 
 $data = App::getFeed();
-$isNewVersionAvailable = isset($data['version']) && $data['version'] != '' && $data['version'] !== Array();
+$isNewVersionAvailable = isset($data['version']) && !empty($data['version']) && $data['version'] !== Array();
```

Comment:
hm. isset is not needed too, manual says `No warning is generated if the variable does not exist.`